Makefile

```diff
@@ -3,11 +3,41 @@ MAINPACKAGE=./cmd/manager
 GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 
+OPERATORNAME := dedicated-admin-operator
+
+# Where do CSVs end up? Note, the CSV generator creates a subdirectory,
+# called $(FINALCSVDIR)
+CSVBASEDIR := csvs
+# FINALCSVDIR is created by the Python script
+FINALCSVDIR := $(CSVBASEDIR)/$(OPERATORNAME)
+
+CATALOG_SOURCEIMG := quay.io/redhat/$(OPERATORNAME)-cs
+
+
 .PHONY: check
 check: ## Lint code
 	gofmt -s -l $(shell go list -f '{{ .Dir }}' ./... ) | grep ".*\.go"; if [ "$$?" = "0" ]; then gofmt -s -d $(shell go list -f '{{ .Dir }}' ./... ); exit 1; fi
 	go vet ./cmd/... ./pkg/...
 
 .PHONY: build
 build: ## Build binary
-	${GOENV} go build ${GOFLAGS} -o ${BINFILE} ${MAINPACKAGE}
\ No newline at end of file
+	${GOENV} go build ${GOFLAGS} -o ${BINFILE} ${MAINPACKAGE}
+
+.PHONY: clean
+clean:
+	rm -rf $(FINALCSVDIR)
```

Comment:
There's a dependency on data in this dir, i.e. `prevver` in `render-csv`.  How is it safe/sane/possible that it can be removed here?